Xamarin.Forms.Controls/TestCases.cs

```diff
@@ -267,7 +268,7 @@ public static NavigationPage GetTestCases ()
 			};
 
 			var searchBar = new SearchBar() {
-				HeightRequest = 42, // Need this for Android N, see https://bugzilla.xamarin.com/show_bug.cgi?id=43975
+				MinimumHeightRequest = 42, // Need this for Android N, see https://bugzilla.xamarin.com/show_bug.cgi?id=43975
```

Comment:
if this is required on Tizen, it probably means the platform is doing something wrong